specification.md

```diff
@@ -344,3 +337,183 @@ In this case, if the `source` value is `source-abc` and the `booking-referral` i
     https://ehr-portal.example.org/bookings?slot=opaque-slot-handle-89172489&source=source-abc&booking-referral=34d1a803-cd6c-4420-9cf5-c5edcc533538
 
 (Note: this construction is *not* as simple as just appending `&source=...` to the booking-deep-link, because the booking-deep-link may or may not already include URL parameters. The Slot Discovery Client must take care to parse the booking-deep-link and append parameters, e.g., including a `?` prefix if not already present.)
+
+
+## Slot Aggregators
+
+Systems that re-publish data from multiple other APIs or from other _Slot Publishers_ are referred to as _Slot Aggregators_. If you are a _Slot Aggregator_, you may wish to use some additional extensions and FHIR features to supply useful provenance information or describe ways that your aggregated data may not exactly match the definitions in the _SMART Scheduling Links_ specification. The practices and approaches in this section are _recommendations_; none are _required_ for a valid implementation.
+
+
+### Preserve Source Identifiers
+
+_Slot Aggregators_ usually need to assign new `id` values to resources in order to make sure they are unique across the aggregated dataset. The `id` of the original resource SHOULD be preserved in the `identifier` list, alongside the identifiers from the original resource. The specific `system` and `value` used for the identifier is left up to the implementer.
+
+For example, given this Location resource from an underlying source:
+
+```js
+{
+  "resourceType": "Location",
+  "id": "123",
+  "identifier": [
+    {
+      "system": "https://cdc.gov/vaccines/programs/vtrcks",
+      "value": "CV1654321"
+    }
+  ],
+  "name": "Flynn's Pharmacy in Pittsfield, MA",
+  // additional Location fields here...
+}
+```
+
+A _Slot Aggregator_ might publish a Location like:
+
+```js
+{
+  "resourceType": "Location",
+  "id": "456",
+  "identifier": [
+    {
+      "system": "https://cdc.gov/vaccines/programs/vtrcks",
+      "value": "CV1654321"
+    },
+    {
+      "system": "https://flynnspharmacy.org/",
```

Comment:
Can you add ".example.org" to any example domains? e.g. "flynnspharamcy.example.org"?

Reply:
Oh, yes, that makes sense. Will do.